plugins/rbac/src/components/CreateRole/AddMembersForm.tsx

```diff
@@ -69,7 +75,7 @@ export const AddMembersForm = ({
       <br />
       <Autocomplete
         options={membersOptions}
-        getOptionLabel={(option: SelectedMember) => option.label}
+        getOptionLabel={(option: SelectedMember) => option.label || ''}
```

Comment:
@debsmita1 is there a scenario where the label can be undefined? If yes then I am not sure showing empty string is the right option here. Maybe in stead of showing empty string we can show the kind or something that will be visible in the drop-down?

Reply:
removed this change